Userland/Libraries/LibWeb/CSS/StyleProperties.cpp

```diff
@@ -135,6 +135,11 @@ RefPtr<CSSStyleValue const> StyleProperties::maybe_null_property(CSS::PropertyID
 CSS::Size StyleProperties::size_value(CSS::PropertyID id) const
 {
     auto value = property(id);
+    // Return none value for global CSS keywords to avoid crash
+    if (value->is_css_wide_keyword()) {
+        return CSS::Size::make_none();
```

Comment:
There is likely a more correct way to handle global CSS keywords but I figured its better to not crash for now 🤷

Reply:
Yeah. Basically we shouldn't be getting those keywords here. The proper solution would be to figure out why animations don't replace those keywords with actual values, and then make them do so. (Which would fix other bugs too.)

Reply:
That makes sense to me. I'll pull on that thread.

Reply:
@AtkinsSJ  Ok. I've finally reached what I think is the end of that thread. It's a little bit hairy but not totally crazy. It's basically described in this [comment](https://github.com/LadybirdBrowser/ladybird/blob/master/Userland/Libraries/LibWeb/Animations/KeyframeEffect.cpp#L879-L882). 

To summarize, the _keyframes_ must be converted to a set of _computed_keyframes_ by resolving the animation styles with the target element.  They are currently not and that's why we end up with the `inherit` property value all the way down in the layout calculation. They must be implemented using the procedure specified in the linked [spec](https://www.w3.org/TR/web-animations-1/#calculating-computed-keyframes). I can take a stab at that, however, it'll take me some time to figure how to reuse/extract the computed style calculations. The other [place](https://github.com/LadybirdBrowser/ladybird/blob/master/Userland/Libraries/LibWeb/CSS/StyleComputer.cpp#L2297) where the style properties are computed has a lot going on there.